api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -119,21 +120,29 @@ public void setUp() {
     DbUser user = new DbUser();
     user.setUsername(USERNAME);
     user = userDao.save(user);
-    providedDbUser = user;
+    dbUser = user;
   }
 
   @Test
-  public void test_updateUserWithRetries_register() {
-    DbUser dbUser = userDao.save(new DbUser());
```

Comment:
no need to create a new user - can use the provided (renamed to dbUser for an easier migration)